app/src/main/res/layout/fragment_main.xml

```diff
@@ -6,39 +6,70 @@
     <androidx.coordinatorlayout.widget.CoordinatorLayout
             android:id="@+id/coord_layout"
             android:layout_width="match_parent"
-            android:layout_height="match_parent">
+            android:layout_height="match_parent"
+            android:background="@color/bgColor">
+
+        <androidx.appcompat.widget.Toolbar
+                android:id="@+id/toolbar"
+                android:layout_width="match_parent"
+                android:layout_height="?attr/actionBarSize"
+                android:background="@color/bgColor"
+                app:menu="@menu/settings">
+
+            <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:orientation="vertical">
+
+                <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginLeft="120dp"
```

Comment:
Can you once also share a screenshot with 90dp as left margin.
I tried the current 120dp version on my device and it was slightly shifted to the right.

Reply:
Left Image: 120dp
Right Image: 90dp

<table>
<tr>
    <td><img src="https://user-images.githubusercontent.com/61552810/95256277-3aa8c400-0840-11eb-8001-b29da537dc63.png" width=250 height=480></td>
<td><img src="https://user-images.githubusercontent.com/61552810/95256284-3c728780-0840-11eb-890b-ba07609a2cb2.png" width=250 height=480></td>
<tr>
</table>

Reply:
Okay. Let it be 120dp then.